### PR TITLE
Use `runIfNot: ...` to avoid running Fuchsia try-jobs for some changes.

### PR DESCRIPTION
**Pull request by matanlurey**
Closes https://github.com/flutter/flutter/issues/93261.

I probably could have been more clever with a regular-expression but this seems easier to read.

### CODE REVIEW

**Comment by matanlurey** [comment]
Hmm, this is failing @godofredoc @zanderso:

```
Protobuf JSON decoding failed at: root["targets"]["5"]["runIfNot"]. Unknown field name 'runIfNot'
runIfNot
```

**Comment by godofredoc** [comment]
> Edit
> Hide
> Delete



> Hmm, this is failing @godofredoc @zanderso:
> 
> ```
> Protobuf JSON decoding failed at: root["targets"]["5"]["runIfNot"]. Unknown field name 'runIfNot'
> runIfNot
> ```

Checking, it seems like I missed updating the proto somewhere.

**Comment by godofredoc** [comment]
> Edit
> Hide
> Delete



> Hmm, this is failing @godofredoc @zanderso:
> 
> ```
> Protobuf JSON decoding failed at: root["targets"]["5"]["runIfNot"]. Unknown field name 'runIfNot'
> runIfNot
> ```

Checking, it seems like I missed updating the proto somewhere.

**Comment by godofredoc** [comment]
Found the issue for the backend deployment https://github.com/flutter/cocoon/pull/3009

**Comment by godofredoc** [comment]
Blocked on https://github.com/flutter/flutter/issues/133376

**Comment by godofredoc** [comment]
@matanlurey this is unblocked now.

**Comment by matanlurey** [comment]
Thank you!!!

File: .ci.yaml [hunk 1]
```diff
@@ -167,6 +167,14 @@ targets:
       # ensure files from pre-production Fuchsia SDK tests are purged from cache
       clobber: "true"
     timeout: 90
+    runIfNot:
```
Comment (zanderso) [inline .ci.yaml:170]: This can also go on the engine_v2 version `Linux linux_fuchsia` down on line 225.

**Review by zanderso — APPROVED** [review]

File: .ci.yaml (hunk 1 above)
Comment (matanlurey) [reply .ci.yaml:170]: Good catch! Re-ordered and added.

**Pull request merged**